plugins/luxtronic2/README.md

```diff
@@ -46,6 +56,12 @@ Defines a mapping to a attribute (read-only). All attribute values are bytes (nu
 ### lux2_c
 Defines a mapping to a calculated value (read-only). All calculated values are integer (numbers).
 
+### lux2_unpack
+Python lambda function. Called before the value is sent towards the device.
+
+### lux2_pack
+Called before the value is written to the smarthome item.
+
```

Comment:
I see your point and I thought about it too. Something like eval_out would work most of the time, but it would fail for something like the example below. My thinking is that a plugin should provide you with a ready to use value and the other way around should take the unencoded value. E.g. for the knx plugin you have to provide knx_dpt and the plugin handles pack/unpack stuff. 
First I only created *_pack for something like below and the way I think of plugins and I did *_unpack out of symmetry. E.g. one could use *_unpack to do the word/byte/... decoding stuff and eval for some value transformation (seconds to days).
I can make the *_pack and *_unpack optional, populated to hand the values through, but I would like to keep them.


```
 [[[dp1]]]
      type = num
      knx_dpt = 9
      knx_send = 10/4/8
      modbus_addr = Blub1 | 1 | 9 | 1
      modbus_type = HoldingRegister
      modbus_readInterval = -1
      modbus_pack = lambda x: [int((x * 1023) / 102.3)]
```